tests/integration/components/truncate-multiline-test.js

```diff
@@ -193,6 +193,64 @@ test('specifying different button texts works', function(assert) {
   );
 });
 
+test('specifying different button a11y texts works', function(assert) {
+  // Template block usage:
```

Comment:
This test is rendering an inline usage, but you can probably just remove this comment.